.github/workflows/e2e.yml

```diff
@@ -77,3 +77,11 @@ jobs:
         uses: grafana/run-k6-action@v1
         with:
           path: 'e2e/script.js'
+
+      - name: Run k6 test claim token api
+        uses: grafana/run-k6-action@v1
+        with:
+          path: 'e2e/test_claim_token_api.js'
+        env:
+          BASE_URL: 'http://127.0.0.1:8787'
+          CALIBNET_RPC: 'https://api.calibration.node.glif.io/rpc/v1'
```

Comment:
why do we need this ?

Reply:
I can remove the ENV from here, but ENV are need in the test script itself.

Reply:
just checking response status 200 for `claim_token` is sufficient here we don't need to verify transaction id and wait for it 
this will also slow the CI checks